### PR TITLE
Cryptomari binance 6

### PR DESCRIPTION
**Pull request by cryptomari0**

### CODE REVIEW

File: db/projects/moeda-loyalty-points/project.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Moeda Loyalty Points
```
Comment (coinmandeer) [inline db/projects/moeda-loyalty-points/project.yaml]: Add full-logo: https://2.bp.blogspot.com/-6SHXfMYVFuU/WnpMTuZOLrI/AAAAAAAABcc/4MDiKsdD3rk3cXD3vUp2AccNJ25eNRdgwCLcBGAs/s1600/moeda%2Bloyalty%2Bpoints%2Brectangle.png

File: db/projects/moeda-loyalty-points/project.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  - seed
+webids:
+  bitcointalk: '2027555.0'
+  telegram: moedaproject
+  github: moedabank
+  twitter: moedaseeds
+  reddit: MoedaBanking
+  facebook: moedaseeds
+  instagram: moedaseeds
+  youtube/channel: 'UCYKNfrimYV-goJP2Du-k7Mw'
```
Comment (coinmandeer) [inline db/projects/moeda-loyalty-points/project.yaml]: Add linkedin: https://www.linkedin.com/company/moeda/

File: db/projects/moeda-loyalty-points/project.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    photo: https://d33wubrfki0l68.cloudfront.net/b2b86c1270b47a3b189840401f3b86ad6c040f72/26c43/assets/images/site/profiles/brad.jpg
+  - name: Isa Yu
+    role: co-founder
+    photo: https://d33wubrfki0l68.cloudfront.net/4f61180a491285ee8061ab436faf6b1380f31c57/0dd65/assets/images/site/profiles/isa.jpg
+  - name: Edward Stehling
+    role: director of operations
+    photo: https://moedaseeds.com/assets/images/site/profiles/edward-stehling.jpg
+  - name: Ubirajara Melo
+    role: cto
+    photo: https://moedaseeds.com/assets/images/site/profiles/ubirajara-melo.jpg
```
Comment (coinmandeer) [inline db/projects/moeda-loyalty-points/project.yaml]: We  should probably focus on any social links of teams, not just photo - we can always get images from their profiles

**Review by coinmandeer-zz — CHANGES_REQUESTED** [review]

File: db/projects/bitcoin-diamond/project.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  - 'http://btcd.io/'
+ledgers:
+  - id: bitcoin-diamond
+    name: Bitcoin Diamond
+    type: blockchain
+    forked: true
+    fork_properties:
+      parent: bitcoin
+      block: 495866
+      time: '2017-11-24T09:20:12Z'
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/bitcoin-diamond/project.yaml]: Yeap, actually see that there is no date:, but **time:**
You can also check schema: ledger: for_properties: time:
and you find format: date-time

I think it is not really necessary for you, but if you are interested, see the link or we can make some personal meeting
https://swagger.io/docs/specification/data-models/data-types/

File: db/projects/bitcoin-diamond/project.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
+name: Bitcoin Diamond
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/bitcoin-diamond/project.yaml]: asset-btd-logo-full.png
asset-btd-logo-square.png

[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/bitcoin-diamond/project.yaml]: asset-btd-logo-alternative.svg

File: db/projects/iotex/project.yaml [hunk 6]
```diff
@@ -0,0 +1,64 @@
+name: IoTeX
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/iotex/project.yaml]: asset-iotx-logo-square.png
asset-iotx-logo-full.png
asset-iotx-logo-alternative.png

File: db/projects/iotex/project.yaml [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  medium: '@iotex'
+  telegram: IoTeXGroup
+  github: iotexproject
+  twitter: iotex_io
+  reddit: IoTex
+  linkedin/company: iotex
+  custom/forum: https://forum.iotex.io/
+contacts:
+  email: 'hello@iotex.io'
+  location:hq: 'Central Region, Singapore'
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/iotex/project.yaml]: location:hq: 'Singapore, Singapore'

File: db/projects/lisk/project.yaml [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+  reddit: lisk
+  gitter: LiskHQ
+  facebook: LiskHQ
+  instagram: liskhq
+  youtube/channel: 'UCuqpGfg_bOQ8Ja4pj811PWg'
+  custom/chat: https://lisk.chat/
+  telegram: 'Lisk_HQ'
+  linkedin/company: lisk
+contacts:
+  location:hq: 'Zug, Switzerland'
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/lisk/project.yaml]: tags:
  - sdk
  - platform
  - framework
  - sidechains
  - dapps
  - nodejs
  - javascript
  - development

[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/lisk/project.yaml]: description: 'Lisk will enable developers to build blockchain applications and deploy their own sidechain linked to the Lisk network, including a custom token. Thanks to the scalability and flexibility of sidechains, developers can implement and customize their blockchain applications entirely.'

File: db/projects/modum/project.yaml [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+      end_date: '2017-09-22'
+      raised: 13350000 USD
+      ico_price: 0.002555 ETH
+      country: Switzerland
+    webids:
+      coinmarketcap: modum
+      cryptocompare: mod
+      cryptocompare/id: '345510'
+team_url: https://modum.io/company/team
+team:
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/modum/project.yaml]: webids:
      linkedin: 'andrew-connell-738a8796'

    webids:
      linkedin: 'philipp-doppelhammer-30544293'
 
    webids:
      linkedin: 'sdoessegger'
 
    webids:
      linkedin: 'marc-heimgartner-245a7a144'
 
    webids:
      linkedin: 'benjamin-indermuehle-1585ba6a'

File: db/projects/moeda-loyalty-points/project.yaml (hunk 1 above)
Comment (cryptomari0) [reply db/projects/moeda-loyalty-points/project.yaml]: asset-mda-logo-full.png

File: db/projects/moeda-loyalty-points/project.yaml (hunk 2 above)
Comment (cryptomari0) [reply db/projects/moeda-loyalty-points/project.yaml]: linkedin/company: moeda

  location:hq: 'Brasilia, Brazil'

File: db/projects/moeda-loyalty-points/project.yaml (hunk 3 above)
Comment (cryptomari0) [reply db/projects/moeda-loyalty-points/project.yaml]: webids:
      linkedin: 'taynaah-reis'
      medium: '@taynaahreis'

    webids:
      linkedin: 'isabella-yu-297212a3'
      medium: '@Isa.moeda'

    webids:
      github: 'ubirajara-infolog'

**Review by coinmandeer-zz — APPROVED** [review]

**Pull request merged**